app/instance.py

```diff
@@ -269,5 +269,53 @@ def ratelimit_handler(error):
         })
 
 
+@app.errorhandler(404)
+def notfound_handler(error):
+    return make_response(json.dumps({
+            'status': 404,
+            'title': 'Not Found'
+        }),
+        404,
+        {
+            'Content-Type': 'application/vnd.api+json'
+        })
+
+
+@app.errorhandler(422)
+def unprocessable_entity_handler(error):
+    return make_response(json.dumps({
+            'status': 422,
+            'title': 'Unprocessable Entity'
+        }),
+        422,
+        {
+            'Content-Type': 'application/vnd.api+json'
+        })
+
+
+@app.errorhandler(400)
+def badrequest_handler(error):
+    return make_response(json.dumps({
+            'status': 400,
+            'title': 'Bad Request'
+        }),
+        400,
+        {
+            'Content-Type': 'application/vnd.api+json'
+        })
+
+
+@app.errorhandler(403)
+def forbidden_error_handler(error):
+    return make_response(json.dumps({
+            'status': 403,
+            'title': 'Unprocessable Entity'
+        }),
+        403,
+        {
+            'Content-Type': 'application/vnd.api+json'
+        })
```

Comment:
Still wrong

Reply:
Can you kindly please tell me @iamareebjamal what changes I need to do? I am registering error handler just as I did before for 429.

Reply:
You can read the issue again

Reply:
Okay